media/css/sandstone/buttons.less

```diff
@@ -402,7 +402,8 @@ ul.download-list {
     display: none !important;
 }
 
-.windows.sha-1 .download-button .os_winsha1 {
+.windows.sha-1 .download-button[data-channel="release"] .os_winsha1,
+.windows.sha-1 .download-button:not([data-channel="release"]) .unsupported-download {
```

Comment:
I'm pretty sure this is going to break older IE's. Support for `not` is patchy at best, I'd rather find a solution that does not rely on this.